tabbycat/options/preferences.py

```diff
@@ -494,16 +494,12 @@ class FeedbackIntroduction(LongStringPreference):
 
 
 @tournament_preferences_registry.register
-class TeamsInDebate(ChoicePreference):
-    help_text = _("Two-team format (e.g. Australs, WSDC) or British Parliamentary")
-    verbose_name = _("Teams in debate")
+class TeamsInDebate(IntegerPreference):
+    help_text = _("How many teams are in each debate (normally 2 for AP or 4 for BP)")
```

Comment:
What does "AP" mean? 😅 

(I'd just say "most debate styles" but maybe that'd be controversial lol)

Reply:
"Associated Press" /jk

Tightened up the wording.